pkg/api/testing/.import-restrictions

```diff
@@ -1,12 +1,7 @@
-{
-	"Rules": [
-	  {
-		"SelectorRegexp": "k8s[.]io/kubernetes/cmd",
-		"AllowedPrefixes": [
-		    "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
-		],
-		"ForbiddenPrefixes": [
-		]
-	  }
-	]
-}
\ No newline at end of file
+rules:
+- selectorRegexp: k8s[.]io/kubernetes
+  allowedPrefixes:
+   - ''
+  forbiddenPrefixes:
+   # prevent pkg/ from depending on cmd/
+   - k8s.io/kubernetes/cmd
```

Comment:
This is redundant, it's already taken care of in `pkg/.import-restrictions`.  Actually, since you're not overriding it anymore, you can remove this file entirely

Reply:
i figured there might be a parent .import-restrictions for that, but didn't check. will update.